rclcpp/include/rclcpp/executors/multi_threaded_executor.hpp

```diff
@@ -61,8 +61,12 @@ class MultiThreadedExecutor : public rclcpp::Executor
   RCLCPP_PUBLIC
   virtual ~MultiThreadedExecutor();
 
+  /// Multi-threaded implementation of spin.
   /**
-   * \sa rclcpp::Executor:spin() for more details
+   * This function will block until work comes in, and try to execute all work with
```

Comment:
This is not correct.
This function won't block until works come in, but rather it will block until interrupted.